docs/rules/max-lines.md

```diff
@@ -0,0 +1,117 @@
+# enforce a maximum file length (max-lines)
+
+Some people consider large files a code smell. Large files tend to do a lot of things and can make following what's going on hard.
+
+## Rule Details
+
+This rule enforces a maximum number of lines per file, in order to aid in maintainability and reduce complexity.
+
+
+## Options
+
+This rule has a number or object option:
+
+* `"max"` (default `100`) enforces a maximum number of lines in a file
```

Comment:
There is no "default" for this rule in JSCS, what mentioned in the docs is just an example.

`100` feels weird to me, i'd say http://www.mind2b.com/component/content/article/24-software-module-size-and-file-size is interesting, my vote for default (if one should exist) is 500